pkg/addons/data/ConfigMap/openshift-node/node-config-compute.yaml

```diff
@@ -21,42 +27,40 @@ data:
       - /etc/origin/node/bootstrap.kubeconfig
       cert-dir:
       - /etc/origin/node/certificates
-      cloud-config:
-      - /etc/origin/cloudprovider/azure.conf
-      cloud-provider:
-      - azure
+      container-runtime:
+      - remote
+      container-runtime-endpoint:
+      - /var/run/crio/crio.sock
       enable-controller-attach-detach:
       - "true"
       feature-gates:
       - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true
+      image-service-endpoint:
+      - /var/run/crio/crio.sock
       node-labels:
       - node-role.kubernetes.io/compute=true
-      - region=primary
       pod-manifest-path:
       - /etc/origin/node/pods
       rotate-certificates:
       - "true"
-      kube-reserved:
-      - "*** GENERATED ***"
-      system-reserved:
-      - "*** GENERATED ***"
+      runtime-request-timeout:
+      - 10m
     masterClientConnectionOverrides:
       acceptContentTypes: application/vnd.kubernetes.protobuf,application/json
-      burst: 200
+      burst: 40
       contentType: application/vnd.kubernetes.protobuf
-      qps: 100
+      qps: 20
     masterKubeConfig: node.kubeconfig
     networkConfig:
-      mtu: 1450
+      mtu: 8951
       networkPluginName: redhat/openshift-ovs-subnet
-    podManifestConfig: null
     proxyArguments:
-      proxy-mode:
-      - iptables
+      cluster-cidr:
+      - 10.1.0.0/16
     servingInfo:
       bindAddress: 0.0.0.0:10250
       bindNetwork: tcp4
-      clientCA: ca.crt
+      clientCA: client-ca.crt
```

Comment:
validate

Reply:
In `master-startup.sh` we have 
`cp /etc/origin/node/ca.crt /etc/origin/node/client-ca.crt`
So this aligns us with upstream